### ISSUE TITLE
exclusive category should be visually demarcated

### ISSUE THREAD

**Issue by andreaschu** [dzhw/zofar#388]
Especially for multiple choice questions an answer option should be clearly distanced if there is the attribute 'exclusive="true"' in qml.
Preferred would be an automated solution against a css snippet in additional css within the template.

**Comment by andreaschu** [comment]
- [ ] needs css class for multiple choice question

also for single choice if there is no pipe (depends on issue #389), maybe in new issue.

**Comment by vdick** [comment]
```
/*
	visually demarcate of exclusive category in multiplechoice questions
*/
.custom-checkbox-exclusive {
	height: 6rem !important;
}
```

**Comment by vdick** [comment]
@SilinaSchirmer to test, you should build the survey again

**Comment by christian-fr** [comment]
@vdick Snippet is working as expected. Can be added to Archetype.

**Comment by vdick** [comment]
@christian-fr  is in archetype implemented, please build a new survey to test it

**Comment by christian-fr** [comment]
CSS-Snippet successfully added to archetype.

**Comment by christian-fr** [comment]
>     * [ ]  needs css class for multiple choice question
> 
> 
> also for single choice if there is no pipe (depends on issue #389), maybe in new issue.

Unclear whether this has been resolved / achieved yet. Issues stays therefore open.

**Comment by vdick** [comment]
@christian-fr  die SingleChoice muss in einem gesonderten Issue aufgenommen werden (haben wir mit @andreaschu abgesprochen)

**Comment by christian-fr** [comment]
Opening up a new issue; closing this one.
